game/scripts/npc/abilities/broodmother_incapacitating_bite_oaa.txt

```diff
@@ -22,7 +22,7 @@
     "RequiredLevel"                                       "6"
     "LevelsBetweenUpgrades"                               "6"
 
-    "AbilityCastRange"                                    "900"
+    "AbilityCastRange"                                    "500"
```

Comment:
I dont think this really needs a nerf, but if you insist I think 500 is too much, put 700 or 800.